src/umpire/interface/c_fortran/typesUmpire.h

```diff
@@ -107,19 +107,19 @@ struct s_umpire_strategy_allocationprefetcher {
 };
 typedef struct s_umpire_strategy_allocationprefetcher umpire_strategy_allocationprefetcher;
 
-// helper capsule_umpire_strategy_dynamicpool
-struct s_umpire_strategy_dynamicpool {
+// helper capsule_umpire_strategy_dynamicpoollist
+struct s_umpire_strategy_dynamicpoollist {
```

Comment:
@kab163 and @davidbeckingsale, I don't think we can do this because it will cause C and Fortran applicaion builds to fail